<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam5 by Santana</string>
  </property>
  <widget class="QPushButton" name="pushButtonCalc">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetOut">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>21</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPDF">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
